PWGCF/EbyEFluctuations/Tasks/Diff_pT_fluct_PID.cxx

```diff
@@ -0,0 +1,758 @@
+    histos.add("fB1_kaon", "", kTProfile2D, {axisCent, axis1Bin});
+    histos.add("fB1_proton", "", kTProfile2D, {axisCent, axis1Bin});
+
+    histos.add("fB2_hadron", "", kTProfile2D, {axisCent, axis1Bin});
+    histos.add("fB2_pion", "", kTProfile2D, {axisCent, axis1Bin});
+    histos.add("fB2_kaon", "", kTProfile2D, {axisCent, axis1Bin});
+    histos.add("fB2_proton", "", kTProfile2D, {axisCent, axis1Bin});
+
+    histos.add("fC_hadron", "", kTProfile2D, {axisCent, axisPt});
+    histos.add("fC_pion", "", kTProfile2D, {axisCent, axisPt});
+    histos.add("fC_kaon", "", kTProfile2D, {axisCent, axisPt});
+    histos.add("fC_proton", "", kTProfile2D, {axisCent, axisPt});
+
+    histos.add("fD_hadron", "", kTProfile2D, {axisCent, axis1Bin});
+    histos.add("fD_pion", "", kTProfile2D, {axisCent, axis1Bin});
+    histos.add("fD_kaon", "", kTProfile2D, {axisCent, axis1Bin});
+    histos.add("fD_proton", "", kTProfile2D, {axisCent, axis1Bin});
+
+    // QA Plots:
+    histos.add("QA/before/h_VtxZ", "V_{Z}", kTH1D, {axisVtxZ});
+    histos.add("QA/before/h_TPCChi2perCluster", "TPC #Chi^{2}/Cluster", kTH1D, {axisChi2});
+    histos.add("QA/before/h_ITSChi2perCluster", "ITS #Chi^{2}/Cluster", kTH1D, {axisChi2});
+    histos.add("QA/before/h_crossedTPC", "Crossed TPC", kTH1D, {axisCrossedTPC});
+    histos.add("QA/before/h2_DcaZ", "DCA_{Z}", kTH1D, {axisDCAz});
+    histos.add("QA/before/h2_DcaXY", "DCA_{XY}", kTH1D, {axisDCAxy});
+    histos.add("QA/before/h2_TPCSignal", "TPC Signal", TPCSignalHist);
+    histos.add("QA/before/h2_TOFSignal", "TOF Signal", TOFSignalHist);
+
+    histos.addClone("QA/before/", "QA/after/");
+
+    histos.add("QA/Pion/h_Pt", "p_{T} (TPC & TPC+TOF)", kTH1D, {axisPt});
+    histos.add("QA/Pion/h_rap", "y (TPC & TPC+TOF)", kTH1D, {axisY});
+    histos.add("QA/Pion/h2_Pt_rap", "p_{T} vs y", kTH2D, {{axisY}, {axisPt}});
+    histos.add("QA/Pion/h2_DcaZ", "DCA_{z}", kTH2D, {{axisPt}, {axisDCAz}});
+    histos.add("QA/Pion/h2_DcaXY", "DCA_{xy}", kTH2D, {{axisPt}, {axisDCAxy}});
+
+    histos.add("QA/Pion/before/h2_TPCNsigma", "n #sigma_{TPC}", TPCnSigmaHist);
+    histos.add("QA/Pion/before/h2_TOFNsigma", "n #sigma_{TOF}", TOFnSigmaHist);
+    histos.add("QA/Pion/before/h2_TpcTofNsigma", "n #sigma_{TPC} vs n #sigma_{TOF}", TPCTOFHist);
+    histos.add("QA/Pion/before/h2_TpcTofNsigma1", "n #sigma_{TPC+TOF}", TPCTOFnSigmaHist);
+
+    histos.add("QA/Pion/h2_TPCNsigma", "n #sigma_{TPC}", TPCnSigmaHist);
+    histos.add("QA/Pion/h2_TOFNsigma", "n #sigma_{TOF}", TOFnSigmaHist);
+    histos.add("QA/Pion/h2_TpcTofNsigma", "n #sigma_{TPC} vs n #sigma_{TOF}", TPCTOFHist);
+    histos.add("QA/Pion/h2_TpcTofNsigma1", "n #sigma_{TPC+TOF}", TPCTOFnSigmaHist);
+
+    histos.add("QA/Pion/h2_TPCSignal", "TPC Signal Pions", TPCSignalHist);
+    histos.add("QA/Pion/h2_TOFSignal", "TOF Signal Pions", TOFSignalHist);
+    histos.add("QA/Pion/h2_ExpTPCSignal", "Expected TPC Signal Pions", TPCSignalHist);
```

Comment:
Have in mind that your titles here will be misleading when you clone them as Kaon and Pions
Just for you to consider it in a later iteration or perhaps in post processing